library/Sexp/src/Juvix/Sexp/Types.hs

```diff
@@ -16,11 +16,13 @@ import qualified Juvix.Library.LineNum as LineNum
 import qualified Juvix.Library.NameSymbol as NameSymbol
 import Prelude (Show (..), String)
 
+type T = Base ()
+
 -- TODO ∷ make Atom generic, and have it conform to an interface?
 -- This way we can erase information later!
```

Comment:
I think this PR allows you to remove this comment, right?

Reply:
Yeah we probably could get rid of that comment